DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -286,9 +286,12 @@
   <class name="pat::Conversion"  ClassVersion="11">
    <version ClassVersion="11" checksum="1239840459"/>
   </class>
-
-  <class name="pat::PackedCandidate" ClassVersion="28">
-    <version ClassVersion="28" checksum="1956349111"/>  
+  <class name="pat::PackedCandidate::PackedCovariance" ClassVersion="10">
+   <version ClassVersion="10" checksum="3320406063"/>
```

Comment:
what's the point of this? we have plenty of 10 for never evolved classes... what's wrong with 10?